addons/l10n_in_website_sale/static/src/js/address.js

```diff
@@ -0,0 +1,71 @@
+import publicWidget from "@web/legacy/js/public/public_widget";
+import { debounce } from "@web/core/utils/timing";
+
+publicWidget.registry.websiteSaleAddressExtended = publicWidget.registry.websiteSaleAddress.extend({
+    events: {
+        'input #o_vat': '_onChangeVat',
+        'change select[name="country_id"]': '_onChangeCountry',
+    },
+
+    /**
+     * @override
+    */
+   init: function() {
+       this._super.apply(this, arguments);
+       this._changeVat = debounce(this._changeVat.bind(this), 400);
+       this.l10n_in_gst_treatment = document.getElementById("l10n_in_gst_treatment");
+       this._gst_treatment = this.addressForm['o_gst_treatment'].value;
+       this._invalid_vat = false;
+    },
+
+    _onChangeVat(ev) {
+       this._changeVat();
+    },
+
+    _changeVat() {
+        this._setL10nInSezVisibility();
+    },
+
+    /**
+     * @override
+     */
+    async _changeCountry(init=false) {
+        this._super();
+        this._setL10nInSezVisibility();
+    },
+
+    _setL10nInSezVisibility() {
```

Comment:
Give a relatable name and as an additional suggestion, you can make the selection disabled too in case user selecting any other country rather than `IN` and make the default to `overseas`. Just make sure the fiscal_country_id is `IN`

Reply:
Hello @sham-odoo 
JS code is not linted properly, Make sure to run the JS linter on your files to follow JS guidelines properly!